### PR TITLE
[OBMIN-363] Search result not found

### PR DESCRIPTION
**Pull request by Voloshkina**
Redesign , created component  search result not found

### CODE REVIEW

File: src/components/search-result/moveCloud/styles.ts [hunk 1]
```diff
@@ -0,0 +1,178 @@
+import styled, { keyframes, css } from 'styled-components';
+
+import { cloud } from 'components/img';
```
Comment (Wolshebnik) [inline src/components/search-result/moveCloud/styles.ts:3]: space

File: src/hooks/useWindowWidth.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { useLayoutEffect, useState } from 'react';
+
+export function useWindowWidth(): number {
+  const [size, setSize] = useState<number>(0);
```
Comment (Wolshebnik) [inline src/hooks/useWindowWidth.ts:4]: space

File: src/hooks/useWindowWidth.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  useLayoutEffect(() => {
+    function updateSize() {
+      setSize(window.innerWidth);
+    }
+    window.addEventListener('resize', updateSize);
+    updateSize();
```
Comment (Wolshebnik) [inline src/hooks/useWindowWidth.ts:10]: space

File: src/hooks/useWindowWidth.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    return () => window.removeEventListener('resize', updateSize);
+  }, []);
```
Comment (Wolshebnik) [inline src/hooks/useWindowWidth.ts:12]: space

**Review by Kyslytsia — APPROVED** [review]

File: src/components/search-result/styles.ts [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    `}
+  `}
+`;
+
+export const StyledImg = styled.img`
+  position: relative;
+  width: 100%;
+  height: 100%;
+  z-index: 15;
+  transform: scale(1.01);
```
Comment (cloudmenson) [inline src/components/search-result/styles.ts:120]: same

**Review by ASGladchenko — APPROVED** [review]

**Review by Viktoria-Serkiz — APPROVED** [review]

File: src/components/search-result/index.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import * as Styles from './styles';
+import { MoveSun } from './moveSun';
+import { MoveCloud } from './moveCloud';
+import { ISearchResult } from './types';
+import { maskCloud } from 'components/img';
```
Comment (Wolshebnik) [inline src/components/search-result/index.tsx:5]: sort
separate absolute paths

File: src/components/search-result/moveSun/index.tsx
Comment (Wolshebnik) [inline src/components/search-result/moveSun/index.tsx:1]: I think this unnecessary

**Review by Wolshebnik — APPROVED** [review]

**Review by cloudmenson — APPROVED** [review]

**Pull request merged**